onecodex/analyses.py

```diff
@@ -239,7 +239,56 @@ def _metadata_fetch(self, metadata_fields, label=None):
 
         return magic_metadata, magic_fields
 
-    def to_df(
+    def to_df(self, analysis_type="classification", **kwargs):
+        """
+        Transform Analyses of samples in a SampleCollection into tabular format.
+
+        Parameters
+        ----------
+        analysis_type : {'classification', 'functional'}, optional
+            The analysis_type to aggregate, corresponding to AnalysisJob.analysis_type
+        kwargs : 'dict', optional
+             Keyword arguments specific to the analysis_type
```

Comment:
I'm worried that this will become unwieldy as we add more `analysis_type`s. Added advice to refer to individual function definitions, do you think that is sufficient?

Reply:
👍